app/views/passengers/index.haml

```diff
@@ -14,6 +14,7 @@
     %label{ for: 'show_inactive' } Show Inactive
     = check_box_tag :show_inactive, true, !@filters.include?('active')
   = submit_tag 'Show Passengers'
+  = submit_tag 'Print This Page', name: "print"
```

Comment:
Small thing: this button doesn't print "this page" it prints what you set the filters to.